src/dev_server.rs

```diff
@@ -11,26 +11,76 @@ use std::{
     process,
 };
 
+/// A simple HTTP server that can, optionally, watch a given command.
+///
+/// Get the files at `watch_path` and serve them at a given IP address
+/// (127.0.0.1:8000 by default). An optional command can be run along the
+/// server, relaunched if changes are detected in your project via a [`Watch`].
+///
+/// # Usage
+///
+/// ```rust,no_run
+/// use std::process;
+/// use xtask_wasm::{
+///     anyhow::Result,
+///     clap,
+///     default_dist_dir,
+/// };
+///
+/// #[derive(clap::Parser)]
+/// enum Opt {
+///     Serve(xtask_wasm::DevServer),
+/// }
+///
+/// fn main() -> Result<()> {
+///     let opt: Opt = clap::Parser::parse();
+///
+///     match opt {
+///         Opt::Serve(mut dev_server) => {
+///             let mut command = process::Command::new("cargo");
+///             command.args(["xtask", "dist"]);
+///
+///             log::info!("Starting the dev server");
+///             dev_server.command(command).start(default_dist_dir(false))?;
+///         }
+///     }
+///
+///     Ok(())
+/// }
+/// ```
+///
+/// Add a `serve` subcommand that will run `cargo xtask dist`, watching for
```

Comment:
Use "start" to match traditional JS dev's server command. https://github.com/facebook/create-react-app#npm-start-or-yarn-start

```suggestion
/// Add a `start` subcommand that will run `cargo xtask dist`, watching for
```